controllers/cluster/cluster_intg_test.go

```diff
@@ -24,10 +24,11 @@ func intgTestEnsureClusterHAProvider() {
 
 	Context("EnsureHAService", func() {
 		var (
-			ctx           *builder.IntegrationTestContext
-			cluster       *clusterv1.Cluster
-			staticCluster *clusterv1.Cluster
+			ctx           		*builder.IntegrationTestContext
```

Comment:
No need to change the whole `cluster_intg_test.go`, since you are moving all logic to another file